src/cryptocom/exchange/private.py

```diff
@@ -314,3 +314,22 @@ async def listen_orders(self, pair: Pair) -> Order:
                 yield Order.create_from_api(
                     self.pairs[data['instrument_name']], order
                 )
+
+    async def create_withdrawal(
+            self, coin: Coin, quantity: str, address: str, client_wid: str = None, address_tag: str = None,
+            network_id: str = None) -> int:
+        """Create withdrawal request."""
+        data = {
+            'currency': coin.name,
+            'amount': quantity,
+            'address': address
+        }
+        if client_wid:
+            data['client_wid'] = str(client_wid)
+        if address_tag:
+            data['address_tag'] = str(address_tag)
+        if network_id:
+            data['network_id'] = str(network_id)
```

Comment:
should be enum

Reply:
Not too sure what you mean? Do you mean that network_ids should be classes?

Reply:
If there are enum "Select the desired network, require the address to be whitelisted first. See default_network and network in get-currency-networks for the value." probably needed to add a generation struct like for pairs

Reply:
will look also at CI something strange 60mi runs, should be timedout by 1-2 mins but they didn't(